uber/templates/preregistration/preregbase.html

```diff
@@ -106,7 +106,7 @@
                 .slice(index, 1 + index)
                 .addClass('active');
             (type.onClick || $.noop)();
-            if (type.extra !== undefined && type.extra !== null) {
+            if (type.extra !== undefined && type.extra !== null && type.extra > $.field('amount_extra').val()) {
```

Comment:
I don't remember offhand whether this would throw an exception for events where kick-ins aren't turned on.  I know that ``$.field('amount_extra')`` will return ``null`` if that field doesn't exist, but I don't recall whether that field is missing or just hidden in cases where ``c.DONATIONS_ENABLED`` is turned off.